crates/languages/src/rust.rs

```diff
@@ -666,6 +678,7 @@ impl ContextProvider for RustContextProvider {
                     RUST_PACKAGE_TASK_VARIABLE.template_value(),
                     format!("--{}", RUST_BIN_KIND_TASK_VARIABLE.template_value()),
                     RUST_BIN_NAME_TASK_VARIABLE.template_value(),
+                    RUST_BIN_REQUIRED_FEATURES_TASK_VARIABLE.template_value(),
```

Comment:
We seem to need `--` before this: 

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/6673c8a0-4363-4560-bf90-a96d54c393c1" />

Reply:
Interesting; I can run all three binaries and the three examples
![Screenshot from 2025-04-09 22-26-45](https://github.com/user-attachments/assets/9fea81a3-4ae8-4a35-a8a2-b859fb133651)

I am testing this on Linux.

Reply:
I'm on mac, now curious what Windows wants for this.

Reply:
Oh, do I need to separate the argument into "--features" and "feature1,feature2"? The first tip in your screenshot says it actually knows the argument  "--features".

I tried that approach in the beginning it's just more ugly, because then we have two optional arguments instead of one.

Reply:
I think that's the culprit, yes, but I'm very surprised Linux gets it right.

Reply:
If that was the issue, it should be fixed now.